### PR TITLE
Adds meeting notes for Feb 2022

### LINKED ISSUE #51: Agenda for 2022-02-16 Meeting (Note the change of date)
**This month's meeting is being moved to 16 February at 16:00 EST, 21:00 UTC.**

### PR DESCRIPTION
**Pull request by rprouse**
Resolves #51

### CODE REVIEW

File: Meetings/2022-02-16=meeting-minutes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Centralizing and providing guidance to project maintainers for common questions/issues that arise and how to address them from [this thread here](https://github.com/dotnet-foundation/Home/discussions/61).
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Introduction of new member Robin Krom
+- Discussed the process to leave the foundation,
+  - Tanner Gooding, it would be good to work with the NuGet team to find process for switching to unsigned packages going forward
+  - How do we deal with it when the original maintainers are no longer available?
```
Comment (glennawatson) [inline Meetings/2022-02-16=meeting-minutes.md:46]: There was discussion in the meeting that your legal department indicated that even if a previous maintainer passed away you'd have to deal with the estate. People may not be available due to backpacking or other travel which is also another consideration.

File: Meetings/2022-02-16=meeting-minutes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - Can we clarify that it is the non-revokable clause that is preventing us from reassigning the rights of the project and add that to the FAQ?
+  - Forking the project and the foundation not continuing to maintain it is an option to leave the foundation.
+  - Is four weeks notice appropriate?
+  - One comment is that the .NET Foundation has been criticized for not communicating and a project leaving might be considered to be a big change
+  - The four week notice gives the perception that the community has influence in the process
```
Comment (glennawatson) [inline Meetings/2022-02-16=meeting-minutes.md:51]: There was mention that this four week notice could be read only and you didn't need to illicit feedback.

File: Meetings/2022-02-16=meeting-minutes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Business that we didn't get to
+
+Briefly discuss ()[https://github.com/dotnet-foundation/wg-maintainers/issues/48]
+
+Talk about how we can provide better guidance to maintainers from this thread as well: [Centralised information for .NET Foundation operation](https://github.com/dotnet-foundation/Home/discussions/61)
+
+## Action Items
+
+- [ ] Rob and Mattias to take the comments on the process back to the board.
+- [ ] Mattias to attempt to get the NuGet team in a meeting about signing packages for teams that leave the foundation and possibly finding a process for switching to unsigned packages.
```
Comment (glennawatson) [inline Meetings/2022-02-16=meeting-minutes.md:72]: There was mention of maybe collecting details of projects who have had active management changes. Possibly getting some sort of agreement with old maintainers to make interactions with the foundation easier.

File: Meetings/2022-02-16=meeting-minutes.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- Mattias Karlsson - @devlead
+- Andrew White - @AndrewTriesToCode
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Michael Hawker
+- Pascal Berger - @pascalberger
+- Robin Krom
+- Tanner Gooding - @tannergooding
```
Comment (glennawatson) [inline Meetings/2022-02-16=meeting-minutes.md:15]: Tom Pappas was in the meeting according to the image capture I made

Comment (rprouse) [reply Meetings/2022-02-16=meeting-minutes.md:15]: Tom is listed below in the Observers section. He works for the foundation.

**Comment by rprouse** [comment]
Changes pushed based on @glennawatson's suggestions. Thanks.

**Review by glennawatson — APPROVED** [review]

**Pull request merged**